pcdscalc/be_lens_calcs.py

```diff
@@ -42,7 +45,7 @@ def configure_defaults(fwhm_unfocused=None, disk_thickness=None,
     Parameters
     -----------
     fwhm_unfocused : float, optional
-        Full width at half maximum unfocused
+        Full width at half maximum unfocused.
```

Comment:
Suggestion to match the unit in the other docstrings
```suggestion
        Full width at half maximum unfocused in meters.
```

Reply:
There are also other elements in the docstring here missing units, though those are from the previous PR